docs/makeown.rst

```diff
@@ -2,11 +2,11 @@ makeown
 =======
 
 .. dfhack-tool::
-    :summary: Converts the selected unit to be a member of your fortress.
+    :summary: Converts the selected unit to be a member of your fortress. Adds them as a citizen.
```

Comment:
a summary can only have one sentence. the description is parsed up to the first period. How about "Converts the selected unit to be a citizen of your fortress"?